contracts/test/v0.8/foundry/vrf/VRFV2PlusWrapper_Migration.t.sol

```diff
@@ -178,21 +189,23 @@ contract VRFV2PlusWrapper_MigrationTest is BaseTest {
       true // check data fields
     );
     address newCoordinatorAddr = address(s_newCoordinator);
-    emit MigrationCompleted(newCoordinatorAddr, subID);
+    emit MigrationCompleted(newCoordinatorAddr, s_wrapperSubscriptionId);
 
-    s_wrapper.migrate(newCoordinatorAddr);
+    // old coordinator has to migrate wrapper's subscription to the new coordinator
+    s_testCoordinator.migrate(s_wrapperSubscriptionId, newCoordinatorAddr);
+    assertEq(address(newCoordinatorAddr), address(s_wrapper.s_vrfCoordinator()));
 
     // subscription no longer exists in v1 coordinator after migration
     vm.expectRevert(SubscriptionAPI.InvalidSubscription.selector);
-    s_testCoordinator.getSubscription(subID);
+    s_testCoordinator.getSubscription(s_wrapperSubscriptionId);
```

Comment:
Could we add a check here to assert that s_wrapper.SUBSCRIPTION_ID() is the newCoordinatorAddr

Reply:
Do you mean the check from line 196? I'm not sure that I understand this comment, `s_wrapper.SUBSCRIPTION_ID()` and `newCoordinatorAddr` are not the same things.

Reply:
Ah my bad, I meant line 196 for the newCoordinatorAddr check, yes.